backend/django/core/models.py

```diff
@@ -235,6 +247,7 @@ class TrainingSet(models.Model):
 
 class RecycleBin(models.Model):
     data = models.ForeignKey("Data")
+    exclude_reason = models.TextField(null=True, default="")
     timestamp = models.DateTimeField(default=timezone.now)
 
 
```

Comment:
This comment isnt related to this line necessarily, but to our models in general.  I think it would be helpful to create custom queryset managers for out models.  That way it is easier to access commonly queries things without having to remember which combination of boolean fields need to be true or false or w/e

Reply:
https://docs.djangoproject.com/en/3.0/topics/db/managers/#creating-a-manager-with-queryset-methods

We also use this pretty extensively on warrants so if you need examples you can check there or ask Peter/myself

Reply:
This is awesome! I was really disliking copying things into filters everywhere because it felt so easy to make a mistake.